safe-resolution-check.js

```diff
@@ -0,0 +1,5 @@
+const { spawn } = require( 'child_process' );
+
+try {
+    spawn( 'npx', ['npm-force-resolutions']);
+} catch(e) {}
```

Comment:
should we maybe log out the error? Or do something with it? Maybe it's confusing for users if we swallow the error and don't give them any feedback.

Reply:
I think logging the report would help. Like Didn't find the package-lock.json. What do you think?

Reply:
@daern91 Added the logging and warning.